src/tdhLoop/tdh.ts

```diff
@@ -540,57 +447,131 @@ export function calculateBoost(
       acquired: 0
     }
   };
+}
 
-  // Category A
-  if (cardSets > 0) {
-    let cardSetBreakdown = 0.25;
-    // additional full sets up to 2
-    cardSetBreakdown += Math.min((cardSets - 1) * 0.02, 0.04);
-    boost += cardSetBreakdown;
-    breakdown.memes_card_sets.acquired = cardSetBreakdown;
+function getHasSeasonSet(
+  seasonId: number,
+  seasons: MemesSeason[],
+  memes: TokenTDH[]
+) {
+  const season = seasons.find((s) => s.id == seasonId);
+  if (!season) {
+    return [];
```

Comment:
This is a strange function. Name implies a boolean result. Then again on some situations you return a boolean and on some you return an array.